File: driver.py
#for token input
import json
#standard imports
import random
#for discord
import asyncio
import discord


#import token file
with open('tokens.json') as file:
  tokens = json.load(file)
file.closed

#start of code discord implementation
client = discord.Client()

@client.event
async def on_ready():
  print('reafy!')

@client.event
async def on_message(message):
  if "mayumi" in message.content.lower():
    if "ping" in message.content.lower():
      await client.send_message(message.channel, 'pong!')
    elif "time" in message.content.lower():
      await client.send_message(message.channel, message.timestamp)

client.run(tokens['discord']) #insert way to find token here.

File: driver.rb
require 'discordrb' #https://github.com/meew0/discordrb
#require 'rubypython' #https://github.com/halostatue/rubypython

#RubyPython.start

bot = Discordrb::Bot.new token: 'MzE2MDg0MTU1MTgyMjE5MjY1.Daf-Wg.L_9s4axCCn2emwqiFOc4ey56pj4'

bot.message(with_text: 'Ping!') do |event|
  event.respond 'Pong!'
end

bot.message(with_text: '8ball') do |event|
  event.respond 'hello' #p 8ball.py.rubify
end

bot.run
